src/Controller/Talk/Create.php

```diff
@@ -0,0 +1,51 @@
+<?php declare(strict_types=1);
+
+namespace App\Controller\Talk;
+
+use App\Dto\TalkRequest;
+use App\Form\TalkType;
+use App\Repository\TalkRepositoryInterface;
+use Symfony\Component\Form\FormFactoryInterface;
+use Symfony\Component\HttpFoundation\RedirectResponse;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\RouterInterface;
+use Twig\Environment as Twig;
+
+final class Create
+{
+    private $renderer;
+    private $formFactory;
+    private $talkRepository;
+    private $router;
+
+    public function __construct(
+        Twig $renderer,
+        FormFactoryInterface $formFactory,
+        TalkRepositoryInterface $talkRepository,
+        RouterInterface $router
+    ) {
+        $this->renderer = $renderer;
+        $this->formFactory = $formFactory;
+        $this->talkRepository = $talkRepository;
+        $this->router = $router;
+    }
+
+    public function handle(Request $request): Response
+    {
+        $talkRequest = new TalkRequest();
```

Comment:
Use Value Objects to avoid empty object. See here : [Value Objects in Symfony Forms](https://webmozart.io/blog/2015/09/09/value-objects-in-symfony-forms/)

Reply:
Fixed :) Object is no longer empty